services/sources/publisher_backlist_service.py

```diff
@@ -4,25 +4,102 @@
 import json
 import urllib.parse
 from typing import Optional
+from model import Record, Work, Edition, Item, Link
 
 from logger import create_log
 from mappings.publisher_backlist import PublisherBacklistMapping
 from managers import S3Manager, WebpubManifest
 from .source_service import SourceService
+from managers import DBManager, ElasticsearchManager
+from elasticsearch_dsl import Search
 
 logger = create_log(__name__)
 
 BASE_URL = "https://api.airtable.com/v0/appBoLf4lMofecGPU/Publisher%20Backlists%20%26%20Collections%20%F0%9F%93%96?view=All%20Lists"
 
 class PublisherBacklistService(SourceService):
     def __init__(self):
+
         self.s3_manager = S3Manager()
         self.s3_manager.createS3Client()
         self.s3_bucket = os.environ['FILE_BUCKET']
         self.prefix = 'manifests/publisher_backlist'
+        self.db_manager = DBManager(
+            user= os.environ.get('POSTGRES_USER', None),
+            pswd= os.environ.get('POSTGRES_PSWD', None),
+            host= os.environ.get('POSTGRES_HOST', None),
+            port= os.environ.get('POSTGRES_PORT', None),
+            db= os.environ.get('POSTGRES_NAME', None)
+        )
```

Comment:
ah last thing is that you can just do DBManager() and it'll init with those default values